src/lib/all_parameter_variant.hpp

```diff
@@ -45,7 +46,7 @@ inline bool is_lqp_column_reference(const AllParameterVariant& variant) {
 }
 
 // Function to check if AllParameterVariant is a placeholder
-inline bool is_placeholder(const AllParameterVariant& variant) { return (variant.type() == typeid(ValuePlaceholder)); }
+inline bool is_parameter(const AllParameterVariant& variant) { return (variant.type() == typeid(ParameterID)); }
```

Comment:
comment still says placeholder. Also, a short comment on what a parameter is would be lovely

Reply:
added a comment to ParameterID declaration